R/get.R

```diff
@@ -0,0 +1,41 @@
+#' Retrieves participants present in secuTrialdata
+#'
+#' Given a secuTrialdata object, this function simply returns a list of participants.
+#' Information included are participant IDs and corresponding study centre information,
+#' if available.
+#'
+#' @param dat secuTrialdata object containing participant IDs and centre information
+#' @return data frame containing a list of participants present in dat
+#' @export
+#'
+#' @examples
+#'
+#' path <- system.file("extdata", "sT_exports", "export_options",
+#'                     "s_export_CSV-xls_CTU05_20191003-144349_all_info.zip",
+#'                     package = "secuTrialR")
+#' sT_export <- read_secuTrial(path)
+#'
+#' # show participants
+#' participants <- get_participants(sT_export)
+#'
+get_participants <- function(dat){
+
```

Comment:
does this need a stop if not secutrialdata?

Reply:
good idea. i'll add it.